Modules/Core/SpatialObjects/test/itkSpatialObjectToImageFilterTest.cxx

```diff
@@ -83,31 +83,69 @@ itkSpatialObjectToImageFilterTest(int, char *[])
   // Testing spacing
   std::cout << "Testing Spacing: ";
 
-  float  spacingFloat[2];
-  double spacingDouble[2];
-
-  for (unsigned int i = 0; i < 2; ++i)
+  constexpr float  floatCheckValue = 1.5;
```

Comment:
Remove from commit message: `# Please enter the commit message for your changes. Lines
starting`